lib/players/computer.rb

```diff
@@ -0,0 +1,83 @@
+module Players
+  class Computer < Player
+    attr_accessor :choice
+
+
+    def move(game)
+      sleep(1)
+      find_move(game)
+      @choice
+    end
+
+    def score(game)
+      if game.winner == "X"
+        return 10
+      elsif game.winner == "O"
+        return -10
+      elsif game.winner == nil
+        return 0
+      end
+    end
+
+    def find_move(game)
+      moves = game.get_available_moves
+
+      @choice = nil
+
+      moves.each do |move|
```

Comment:
It seems that you repeat the same exact logic for both `"X"` and `"O"` which is not very 'DRY'
It might be a good idea to abstract this to a method e.g.
``` ruby
def make_choice(game, move, token)
  game.board.cells[move - 1]  = token
  winner = (game.winner == token)
  game.board.cells[move - 1] = " "
  winner || nil
end
```

`@choice ||= (make_choice(game, move, "X") || make_choice(game, move, "O")`

(^ This is all just a quick rough idea - you could implement this however you want)